libs/feature-install-node-js/src/lib/install-node-js.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Apollo } from 'apollo-angular';
+import gql from 'graphql-tag';
+import { Observable } from 'rxjs';
+import { map, switchMap, tap } from 'rxjs/operators';
+
+import { InstallNodeJsStatus } from '../../../../server/src/graphql-types';
```

Comment:
could you remove this and replace this with any for now. I don't think the client-side code should reference the server folder.

Once nx 6.4 is out, I'm going to upgrade console to use it, so we can share libs between the server and the client. Then we can share all the types.

Reply:
Donezo